packages/@aws-cdk/custom-resources/lib/provider-framework/runtime/framework.ts

```diff
@@ -109,6 +117,23 @@ async function invokeUserFunction(functionArnEnv: string, payload: any) {
 
   const jsonPayload = parseJsonPayload(resp.Payload);
   if (resp.FunctionError) {
+    if (resp.FunctionError.includes('Lambda is initializing your function')) {
```

Comment:
There should be an error code. In fact, doesn't the `invokeFunction` call above just straight up fail with an exception?

Reply:
Yes, there should be! I don't know what it is yet though, so I will add logic for it when I figure out the exact error code

Reply:
Do you have a plan for finding out what it is? 😀

(Iow, how are you going to achieve that?)